### PR TITLE
[DOCS-9163]Network path setup for ECS

### PR DESCRIPTION
**Pull request by aliciascott**
### What does this PR do? What is the motivation?
[DOCS-9163](https://datadoghq.atlassian.net/browse/DOCS-9163)
Updates setup instructions on the Amazon ECS page for Network Path, adding "private beta" label as well to match Network Path docs for the time-being.

### Merge instructions

- [x] Please merge after reviewing

### Additional notes

[DOCS-9163]: https://datadoghq.atlassian.net/browse/DOCS-9163?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by AlexandreYang — APPROVED** [review]

File: content/en/containers/amazon_ecs/_index.md [hunk 1]
```diff
@@ -261,7 +261,49 @@ If you already have a task definition, update your file to include the following
    "family": "datadog-agent-task"
  }
  ```
+#### Network Path
 
+<div class="alert alert-info">Network Path for Datadog Network Performance Monitoring is in private beta. Reach out to your Datadog representative to sign up.</div>
```
Comment (brett0000FF) [inline content/en/containers/amazon_ecs/_index.md:266]:
```suggestion
<div class="alert alert-info">Network Path for Datadog Network Performance Monitoring is in Preview. Reach out to your Datadog representative to sign up.</div>
```
Should this be "in Preview" now?

File: content/en/containers/amazon_ecs/_index.md [hunk 2]
```diff
@@ -261,7 +261,49 @@ If you already have a task definition, update your file to include the following
...
+          "value": "true"
+        }
+      ],
+   ```
+
+2. To monitor individual paths, follow the instructions here to [set up additional Agent features](#set-up-additional-agent-features):
+
+   These files deploy an Agent container with a base configuration to collect core metrics about the containers in your ECS cluster. The Agent can also run Agent integrations based on Docker Labels discovered on your containers.
+
+3. To monitor network traffic paths to allow the Agent to automatically discover and monitor network paths based on actual network traffic, without requiring you to specify endpoints manually, add the following additional environment variables to your `datadog-agent-sysprobe-ecs.json`:
```
Comment (brett0000FF) [inline content/en/containers/amazon_ecs/_index.md:284]:
```suggestion
3. To monitor network traffic paths and allow the Agent to automatically discover and monitor network paths based on actual network traffic, without requiring you to specify endpoints manually, add the following additional environment variables to your `datadog-agent-sysprobe-ecs.json`:
```

**Review by brett0000FF — APPROVED** [review]
Two minor suggestions!

File: content/en/containers/amazon_ecs/_index.md (hunk 1 above)
Comment (aliciascott) [reply content/en/containers/amazon_ecs/_index.md:266]: Hi @brett0000FF not yet for this one. The app still says "Alpha" as well as the main Network Path docs still say "private beta". Targeting Limited Availability for re:invent.. Let me check.

**Pull request merged**